org.hl7.fhir.r4/src/main/java/org/hl7/fhir/r4/utils/client/FHIRToolingClient.java

```diff
@@ -188,7 +201,13 @@ public CapabilityStatement getCapabilitiesStatementQuick() throws EFhirClientExc
     try {
       capabilities = (CapabilityStatement)utils.issueGetResourceRequest(resourceAddress.resolveMetadataUri(true), getPreferredResourceFormat(), "CapabilitiesStatement-Quick", TIMEOUT_NORMAL).getReference();
     } catch(Exception e) {
-      handleException("An error has occurred while trying to fetch the server's conformance statement", e);
+        // THIS IS A WORKAROUND FOR THE TERMINOLOGY SERVER RESPONDING WITH A 404 TO /metadata
+		capabilities = new CapabilityStatement();
+		capabilities.setStatus(Enumerations.PublicationStatus.ACTIVE);
+		capabilities.setDateElement(DateTimeType.now());
+		capabilities.setKind(CapabilityStatement.CapabilityStatementKind.INSTANCE);
+		capabilities.setFhirVersion(Enumerations.FHIRVersion._4_0_1);
+		//handleException("An error has occurred while trying to fetch the server's conformance statement", e);
```

Comment:
If the call is returning a 404, we need to deal with the 404 and not put code in the `catch` block to execute. This is not good practice.